src/index.js

```diff
@@ -810,6 +830,13 @@ class EdfContentScript extends ContentScript {
         // is ready to receive it
         this.store = { email, password }
       }
+    } else if (event === 'interceptedResponse') {
+      // store edf token token, intercepted in xhr response to use it when calling edf api
+      this.log('debug', 'intercepted new edf token')
+      this.csrfToken = payload?.response?.data
+      if (!this.csrfToken) {
+        this.log('error', 'Wrong edf token intercepted: ', this.csrfToken)
```

Comment:
a cet endroit `this.csrfToken` sera toujours null ou undefined vu le if juste au dessus. C'est peut-être plus payload.response que tu veux loguer?